src/content/docs/guides/JSON/0-json_intro.mdx

```diff
@@ -89,6 +89,28 @@ This command creates sub-folders for each type of resource. One of these is name
   ```
 
   </TabItem>
+  <TabItem label="Python">
+
+```python
+from splashkit import *
+
+def main():
+    game_data = json_from_file("game_data.json")
+    game_title = json_read_string(game_data, "gameTitle")
+    
+    if not game_title:
+        print("Error retrieving gameTitle.")
+    else:
+        print("Game Title:", game_title)
```

Comment:
This needs to use SplashKit's "write_line" function to follow the same format as the other code.